Update us.m3u

freearhey opened the pull request:
Closes https://github.com/iptv-org/iptv/issues/14470, closes #14482

freearhey committed:
Update us.m3u

freearhey committed:
Update us_local.m3u

freearhey committed:
Update us_pbs.m3u

BellezaEmporium commented:
Only https://github.com/iptv-org/iptv/blob/d526474567002f0e121290fbbb250b9c867b3ea9/streams/us_pbs.m3u#L124-L143 works.

freearhey committed:
Update us_pbs.m3u

freearhey commented:
Updated.

BellezaEmporium commented on streams/us_pbs.m3u:
```diff
@@ -141,147 +19,3 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
-#EXTINF:-1 tvg-id="WCMUDT1.us",PBS Mount Pleasant MI (WCMU) (1080p)
-https://wcmudt.lls.pbs.org/out/v1/45ff0524571c41398c5f39ebab6262ef/index.m3u8
-#EXTINF:-1 tvg-id="WMPTDT1.us",PBS MPT Baltimore MD (WMPB) (1080p)
-https://2-fss-2.streamhoster.com/pl_138/amlst:201814-1291584/playlist.m3u8
```
This one works too. My bad.

BellezaEmporium commented on streams/us_pbs.m3u:
```diff
@@ -141,147 +19,3 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
-#EXTINF:-1 tvg-id="WNPTDT1.us",PBS Nashville TN (WNPT) (1080p)
-https://wnptdt.lls.pbs.org/out/v1/f6bca6d722674c5cad621f54b17c217b/index.m3u8
-#EXTINF:-1 tvg-id="PBSEast.us",PBS National East (1080p) [Geo-blocked]
-https://pbs.lls.cdn.pbs.org/est/index.m3u8
```
This one works too. My bad.

BellezaEmporium commented on streams/us_pbs.m3u:
```diff
@@ -141,147 +19,3 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
-#EXTINF:-1 tvg-id="PBSEast.us",PBS National East (1080p) [Geo-blocked]
-https://pbs.lls.cdn.pbs.org/est/index.m3u8
-#EXTINF:-1 tvg-id="PBSWest.us",PBS National West (1080p) [Geo-blocked]
-https://pbs.lls.cdn.pbs.org/pst/index.m3u8
```
This one works too. My bad.

BellezaEmporium commented on streams/us_pbs.m3u:
```diff
@@ -141,147 +19,3 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
-#EXTINF:-1 tvg-id="KPTSDT1.us",PBS Wichita KS (KPTS) (1080p)
-https://kptsdt.lls.pbs.org/out/v1/1ca3404d6bc3404c9daa86c8f1ae19d0/index.m3u8
-#EXTINF:-1 tvg-id="WMPTDT2.us",WMPT-DT2 (PBS Create)
-https://2-fss-1.streamhoster.com/pl_122/amlst:201814-2122456/playlist.m3u8
```
This one works too. My bad.

BellezaEmporium reviewed (commented):
Found out HLS links works... if you match the IP to the PBS station. How odd. I remember they used to work everywhere as long as you had an US IP. Now it's even more restrictive.

freearhey committed:
Update us_pbs.m3u

freearhey commented:
Updated.

BellezaEmporium reviewed (dismissed):
Reserved approval.

While we're updating US channels : I have a few here : 

https://wway.akamaized.net/hls/live/2029583/wway_ingest/master.m3u8 ABC WWAY 3 (WWAY-DT1) Wilmington, North Carolina
https://vbfast-c.viebit.com/810fe13b-a9ac-46b3-a523-5a7e90ddfd25/playlist.m3u8 Access La Porte Country Channel 97 Michigan City, Indiana
https://livestream.telvue.com/nashuanh1/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 Access Nashua Nashua, New Hampshire
https://livestream.telvue.com/nashuanh2/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 Nashua ETV Channel 22 Nashua, New Hampshire
https://livestream.telvue.com/nashuanh3/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 Nashua GTV Channel 16 Nashua, New Hampshire
https://livestream.telvue.com/accvision1/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 AccessVision Channel 16 Battle Creek, Michigan
https://livestream.telvue.com/accvision2/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 AccessVision Channel 17 Battle Creek, Michigan

BellezaEmporium commented:
Asking @PopeyeTheSai10r here to verify the theory I have over the PBS HLS links.

freearhey committed:
Update us.m3u

Source: https://github.com/iptv-org/iptv/pull/14483#pullrequestreview-1691263607

freearhey committed:
Merge branch 'master' into update-us-m3u

freearhey commented:
Added.

PopeyeTheSai10r commented:
> Asking @PopeyeTheSai10r here to verify the theory I have over the PBS HLS links.

Some of the streams/us_pbs.m3u links are still working but they seen to be region-locked. 

The following links worked for me while using a NY IP. They won't work when using US IP from other states.
```
https://wnetdt.lls.pbs.org/out/v1/0456457548354b88b32fc437e4e7ee01/index.m3u8
https://wnjtdt.lls.pbs.org/out/v1/e62efd8d4f92403996425fc389df0ffd/index.m3u8
```

freearhey committed:
Update us_pbs.m3u

https://github.com/iptv-org/iptv/pull/14483#issuecomment-1773986794

freearhey commented:
> The following links worked for me while using a NY IP. They won't work when using US IP from other states.
>
> https://wnetdt.lls.pbs.org/out/v1/0456457548354b88b32fc437e4e7ee01/index.m3u8
> https://wnjtdt.lls.pbs.org/out/v1/e62efd8d4f92403996425fc389df0ffd/index.m3u8

Brought those links back.

BellezaEmporium commented:
I'll check with my VFN over the available regions and come back with an answer.

BellezaEmporium commented on streams/us.m3u:
```diff
@@ -1139,4 +1133,18 @@ http://190.11.225.124:5000/live/playboy_hd/playlist.m3u8
 http://51.159.102.100/PenthouseGold/index.m3u8
 #EXTINF:-1 tvg-id="BrazzersTVEurope.us",Brazzers TV Europe (720p)
-https://cdn-main.lolokoko.tv/BRAZZERS.stream/chunks.m3u8
+https://cdn-global.lolokoko.tv/BRAZZERS.stream/chunks.m3u8
```
I believe that wasn't intentional.

freearhey replied:
Updated.

BellezaEmporium commented on streams/us_pbs.m3u:
```diff
@@ -1,128 +1,4 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="KETSDT1.us",Arkansas PBS [Geo-blocked]
-https://ketsdt.lls.pbs.org/out/v1/03c094dbd7874a4a8c3fe9fb10081bdb/index.m3u8
```
Works (VPN set to Buffalo & New York)

BellezaEmporium commented on streams/us_pbs.m3u:
```diff
@@ -1,128 +1,4 @@
-#EXTINF:-1 tvg-id="WEDHDT1.us",PBS Hartford CT (WEDH) (1080p)
-https://wedhdt.lls.pbs.org/out/v1/04182c8d6be24c1a98de212f3c55a442/index.m3u8
-#EXTINF:-1 tvg-id="KHETDT1.us",PBS Honolulu HI (KHET) (1080p) [Not 24/7]
-https://khetdt.lls.pbs.org/out/v1/7ec7903413294b72bb64f83963d8ea9b/index.m3u8
```
Works (VPN set to Buffalo & New York)

BellezaEmporium commented on streams/us_pbs.m3u:
```diff
@@ -143,147 +19,15 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
-#EXTINF:-1 tvg-id="WLJTDT1.us",PBS Lexington TN (WLJT) (1080p)
-https://wljtdt.lls.pbs.org/out/v1/1ddf810d67f64eeeab40ad9da8885945/index.m3u8
-#EXTINF:-1 tvg-id="KUONDT1.us",PBS Lincoln NE (KUON) (1080p) [Geo-blocked]
-https://kuondt.lls.pbs.org/out/v1/91d8b5ffc5c1453c8a621508a07749a6/index.m3u8
```
Works (VPN set to Buffalo & New York)

freearhey committed:
Update us.m3u

freearhey committed:
Update us_pbs.m3u

freearhey commented:
Updated.
